tests/features/laco-widget.feature

```diff
@@ -0,0 +1,22 @@
+@api
+Feature: Webtools LACO Widget
+  In order to provide LACO
+  As the site manager
+  I need to be able to configure LACO widget
+  And LACO widget works as expected
+
+  Background:
+    Given I am logged in as a user with the "administer site configuration" permission
+
+  @BackupLacoConfigs
+  Scenario: Create Webtools Analytics Rule
```

Comment:
Is it a rule that you are making or setting a configuration?

Reply:
Ops, it's configuration, of course. Fixed.